config/database.go

```diff
@@ -117,7 +114,26 @@ func initializeConfigurationsTable(db *sqlx.DB) error {
 		if err != nil {
 			return errors.Wrap(err, "failed to alter Configurations table character set")
 		}
+	}
 
+	err = db.Get(&count, `SELECT COUNT(*) FROM `+migrationsTableName+` WHERE Name = 'create_configuration_files'`)
```

Comment:
Shouldn't this be split into 2 functions? If the previous migration is done, then it will skip checking this one.

Reply:
Right, we need to check both, if the 1. check for `create_configurations` completed we are skipping checking the `create_configuration_files`. Nice catch 🙏